src/test/java/org/broadinstitute/hellbender/utils/SimpleIntervalUnitTest.java

```diff
@@ -208,6 +208,26 @@ public void testOverlapWithMargin( final SimpleInterval firstInterval, final Sim
                 "overlap() returned incorrect result for intervals " + firstInterval + " and " + secondInterval);
     }
 
+    @DataProvider(name = "overlapsWithMarginExpectingException")
+    private Object[][] overlapsWithMarginExpectingException() {
+        final SimpleInterval standardInterval = new SimpleInterval("1", 10, 20);
+        final SimpleInterval middleInterval = new SimpleInterval("1", 100, 200);
+
+        return new Object[][] {
+                { standardInterval, new SimpleInterval("2", 10, 20), -100, false },
+                { standardInterval, new SimpleInterval("1", 30, 50), -9, false },
+                { standardInterval, new SimpleInterval("1", 30, 50), -10, true },
+                { middleInterval, new SimpleInterval("1", 50, 90), -9, false },
+                { middleInterval, new SimpleInterval("1", 50, 90), -10, true },
```

Comment:
There isn't actually an expected result for these test cases, since you expect them all to throw, so you can omit the final `expectedOverlapResult` boolean, and just call `firstInterval.overlapsWithMargin(secondInterval, margin);` without the assert in the body of the test method.

Reply:
done. Thanks for catching that quick copy-and-paste error.